### DISCUSSION TITLE
Request for decrease of Max Level cap to 30 in response to player culture

### DISCUSSION THREAD

**Discussion by CrankyLandlord** [LBPUnion/ProjectLighthouse#978] [Ideas]
I base my assessment on my experience on Beacon, but am confident that the patterns that I see could be observed from any other instance of Lighthouse if the sample size is large enough.

When you really look at the very few players with higher level counts, we’ll say over 30, generally they fall in one of three categories: massive archive accounts, very prolific creators (mostly with reuploads of pre-shutdown creations), or players with incredibly low effort joke levels/default moon bg littered with pre-made object type levels. I’m all for dishing out extra slots to archivers or actual creators, but we shouldn’t automatically trust just any user with a bunch of extra storage which most people don’t even need. Because the sky's the limit, low-effort level creators will never have to consider taking down their levels to make room for new ones, the server and the newest level section will continually be clogged up by garbage.

**Comment by sudokoko** [comment]
Hello! Thank you for your suggestion.

The maximum level cap is configurable by server instance administrators. I've attached the code snippet below which indicates the configuration option.

https://github.com/LBPUnion/ProjectLighthouse/blob/983b1e998472a9630f2dfb385a31b627c4b0a031/ProjectLighthouse/Configuration/ConfigurationCategories/UserGeneratedContentLimitConfiguration.cs#L8

In terms of Beacon, you will need to take this up with the Starguard Captain as well as the Star Commanders, as they oversee the administration of Beacon from a front-end perspective.
